src/main/java/com/padmaja/inheritance/Test.java

```diff
@@ -0,0 +1,18 @@
+package com.padmaja.inheritance;//Java program to illustrate the
+// concept of inheritance 
+
+
+
+
+
+// driver class 
+public class Test 
+{ 
+	public static void main(String args[]) 
+	{ 
+		
+		MountainBike mb = new MountainBike(3, 100, 25); 
```

Comment:
try create a object from base class not from MountainBike.
And see what it prints.
Bicycle b = new MountainBike(3, 100, 25);
b.toString()